addons/website_jitsi/static/src/js/chat_room.js

```diff
@@ -44,6 +44,8 @@ publicWidget.registry.ChatRoom = publicWidget.Widget.extend({
 
         this.jitsiServer = this.$el.data('jitsi-server') || 'meet.jit.si';
 
+        this.maxCapacity = parseInt(this.$el.data('max-capacity')) || Infinity;
```

Comment:
Infinity ? :octopus:

Reply:
To avoid additional condition bellow (so when the value is 'no_limit' maxCapacity will be "Infinity")

Reply:
My remark was about Infinity itself ... did not know it existed . Does JS handle abstract concepts, like Life, Death or RunningZombies ?

Reply:
The most weird keyword I know is Ellipsis in python 
```python
>>> Ellipsis == ...
True
```